src/include/api.h

```diff
@@ -253,6 +253,15 @@
     if (F_ISSET(cur, WT_CURSTD_CACHED))                                                    \
     WT_ERR(__wt_cursor_cached(cur))
 
+#define API_RETRYABLE(s) do {
```

Comment:
I think both of these could use a small comment about their purpose and use.

Reply:
I've added a comment, including some notes about the placement.